app/src/main/kotlin/com/simplemobiletools/keyboard/dialogs/ChangeLanguagePopup.kt

```diff
@@ -0,0 +1,56 @@
+package com.simplemobiletools.keyboard.dialogs
+
+import android.view.View
+import com.simplemobiletools.commons.models.RadioItem
+import com.simplemobiletools.keyboard.R
+import com.simplemobiletools.keyboard.extensions.config
+import com.simplemobiletools.keyboard.helpers.*
+
+class ChangeLanguagePopup(
+    inputView: View,
+    private val onSelect: () -> Unit,
+) {
+    private val context = inputView.context
+    private val config = context.config
+
+    init {
+        val items = arrayListOf(
+            RadioItem(LANGUAGE_BENGALI, getKeyboardLanguageText(LANGUAGE_BENGALI)),
+            RadioItem(LANGUAGE_BULGARIAN, getKeyboardLanguageText(LANGUAGE_BULGARIAN)),
+            RadioItem(LANGUAGE_ENGLISH_QWERTY, getKeyboardLanguageText(LANGUAGE_ENGLISH_QWERTY)),
+            RadioItem(LANGUAGE_ENGLISH_QWERTZ, getKeyboardLanguageText(LANGUAGE_ENGLISH_QWERTZ)),
+            RadioItem(LANGUAGE_ENGLISH_DVORAK, getKeyboardLanguageText(LANGUAGE_ENGLISH_DVORAK)),
+            RadioItem(LANGUAGE_FRENCH, getKeyboardLanguageText(LANGUAGE_FRENCH)),
+            RadioItem(LANGUAGE_GERMAN, getKeyboardLanguageText(LANGUAGE_GERMAN)),
+            RadioItem(LANGUAGE_LITHUANIAN, getKeyboardLanguageText(LANGUAGE_LITHUANIAN)),
+            RadioItem(LANGUAGE_ROMANIAN, getKeyboardLanguageText(LANGUAGE_ROMANIAN)),
+            RadioItem(LANGUAGE_RUSSIAN, getKeyboardLanguageText(LANGUAGE_RUSSIAN)),
+            RadioItem(LANGUAGE_SLOVENIAN, getKeyboardLanguageText(LANGUAGE_SLOVENIAN)),
+            RadioItem(LANGUAGE_SPANISH, getKeyboardLanguageText(LANGUAGE_SPANISH)),
+            RadioItem(LANGUAGE_TURKISH_Q, getKeyboardLanguageText(LANGUAGE_TURKISH_Q))
```

Comment:
lets move these items and getKeyboardLanguageText() to Constants.kt or somewhere to avoid duplication as it is used in SettingsActivity too. It is too error prone having it all on 2 places as we might forget about one in the future, when adding new languages.

Reply:
Moved it to the `extensions/Context.kt` file